src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Selection.js

```diff
@@ -78,6 +79,42 @@ export default class Selection extends React.Component<Props> {
         }
     }
 
+    componentDidUpdate(prevProps: Props) {
```

Comment:
This function definitely needs a test, especially because this code probably has no effect in the core system :see_no_evil:

Reply:
This still holds true :slightly_smiling_face: